docker/serve/Dockerfile

```diff
@@ -19,21 +30,20 @@ RUN apt-get update && \
     ffmpeg libsm6 libxext6 git ninja-build libglib2.0-0 libsm6 libxrender-dev libxext6 \
     && rm -rf /var/lib/apt/lists/*
 
-ENV PATH="/opt/conda/bin:$PATH"
-RUN export FORCE_CUDA=1
+ENV PATH="/opt/conda/bin:$PATH" \
+    FORCE_CUDA="1"
 
 # TORCHSEVER
 RUN pip install torchserve torch-model-archiver
 
 # MMLAB
 ARG PYTORCH
 ARG CUDA
+RUN pip install mmengine
 RUN ["/bin/bash", "-c", "pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html"]
```

Comment:
Can we also use mim to install mmcv in this dockerfile?

Reply:
updated